packages/tab/index.tsx

```diff
@@ -142,27 +202,19 @@ export default class Tab extends Component {
       isMinWidthIndicator,
       ...otherProps
     } = this.props;
-
-    const {
-      tabIndex,
-      ['aria-selected']: ariaSelected,
-    } = this.state;
-
+    const {tabIndex, ['aria-selected']: ariaSelected} = this.state;
```

Comment:
Why did you collapse these?

Personally, I find vertically-delimited lists much easier to maintain over time.